### ISSUE TITLE
Recipes

### ISSUE THREAD

**Issue by agriffard** [AidosMarcos/OrchardTheme#1]
You could create recipes for each theme:

- Create a deployment plan and add steps to export all the content types, the contents and a custom file. It will allow you to obtain the corresponding recipe json file.
- Configure it and add it to a Recipes folder at the root of your theme.

It will be available in the list of recipes when you setup a new site.

You can take example on the Resume theme I made: https://github.com/agriffard/Resume.OrchardCore

You could also split the themes in 2 different repositories and if you provide them with a MIT license, you could use a free appveyor account to build your theme and upload it to NuGet.

**Comment by AidosMarcos** [comment]
Thank you for taking the time to look at this.

Yes, i will do that, will create a recipe for each of the Themes.

My first goal at doing this was really understand how everything worked, without expecting to received such good feedback, so i will keep working on this and maybe make some more.

**Comment by agriffard** [comment]
All your work will make more sense if: 
- You make one repository for each theme. 
- You create the corresponding recipe that will setup the content types and default content items.
- You publish a package on Nuget.

You made an OC application using OrchardCore.Cms.Targets, with a sqlite db and you committed all the appllication. 

I understand why you did it like this (it was easier for you and it was a test to lean how themes are made) but to test it, I cannot guess the admin user/password that you used (and I don't suggest you to give it).

Even if someone wants to test it directly with a project reference, it is easier to clone a repository that contains only one theme.

**Comment by psijkof** [comment]
@agriffard Together with Marcos we created a netstandard class lib and added the recipe and using your example other missing files. The theme is recognized by the oc cms (sample) web app, but fails to install the theme with error "JavaScriptException: uuid is not defined (Jint.Engine.Execute(Program program))
Tried on two different machines to rule out any package version mismatch.
Any idea what we are missing?
Using version 1.0.0-beta3-68220 of OC via preview myget feed.
